sdk/README.md

```diff
@@ -50,7 +50,7 @@ SDK provides a `TektonCompiler` and a `TektonClient`:
 ## Project Prerequisites
 
  - Python: `3.5.3` or later
- - Tekton: [`0.14.0`](https://github.com/tektoncd/pipeline/releases/tag/v0.14.0)
+ - Tekton: [`0.14.0`](https://github.com/tektoncd/pipeline/releases/tag/v0.15.0) or later
```

Comment:
Maybe we should have the release page instead of pointing to a certain tag?

Reply:
I want people who already have a Tekton installation to find that their version 0.14 is okay to use, or, if they don't have an installation yet, to follow the link and get the latest version we tested with (0.15.0 now)

Reply:
this is confusing though - 0.14 hyperlinked to 0.15 tag. 
Can we say something like supported versions: 0.14 and 0.15, with each hyperlinked to their respective release tags...?

Reply:
How about:

**Project Prerequisites**
- ...
- Tekton: [`v0.14.0`](https://github.com/tektoncd/pipeline/releases/tag/v0.14.0) or [later](https://github.com/tektoncd/pipeline/releases/latest)
- ...

Reply:
I tested all of these version, btw:

```YAML
v0.14.0: OK
v0.14.1: OK
v0.14.2: OK
v0.14.3: OK
v0.15.0: OK
v0.15.1: OK
```